SpokeStack/WebRTCVAD.swift

```diff
@@ -9,27 +9,16 @@
 import Foundation
 import filter_audio
 
-/// Indicate to the VAD the level of permissiveness to non-speech activation.
+/// Indicate how likely it is that non-speech will activate the VAD.
 public enum VADMode: Int {
-    /// Most permissive of non-speech; least amount of missed speech detection
-    case HighlyPermissive
-    /// Allows more non-speech
-    case Permissive
-    /// Allows less non-speech
-    case Restrictive
-    /// Most restrictive of non-speech; most amount of missed speech detection
-    case HighlyRestrictive
-}
-
-/// VAD speech activation confidence level
-public enum VADDecision: Int {
-    case None
-    case Uncertain
-    case Possible
-    case Low
-    case Medium
-    case High
-    case Highest
+    /// Most permissive of non-speech; least likely to not detection speech; most likely to detect speech.
+    case HighlyPermissive = 1
+    /// Allows more non-speech than .higher levels.
```

Comment:
"higher"..also, "than higher levels" might not be a useful clarification; it's really "more" that's the problem, but I know that's hard/impossible to quantify.